index.test.js

```diff
@@ -755,7 +755,7 @@ describe('Deploy to ECS', () => {
             cluster: 'cluster-789',
             service: 'service-456',
             taskDefinition: 'task:def:arn',
-            forceNewDeployment: true
+            forceNewDeployment: false
```

Comment:
the name of this test is force new deployment, but you're giving it a value of false (shouldn't this be false by default/wouldn't you want to test true?)